CONTRIBUTING.md

```diff
@@ -244,12 +244,29 @@ This workflow then files a PR to `master` with the rendered and live files.
 
 ## Cheatsheets
 
-Training modules have corresponding cheatsheets in `module-cheatsheets`. 
-Cheatsheets are written in plain markdown and can be converted to a shareable PDF format interactively as follows:
+Training modules have corresponding cheatsheets in `module-cheatsheets`.
+When choosing documentation links to incorporate in cheatsheets, we prefer to use [`https://rdrr.io/`](https://rdrr.io/) when possible for Base R and Bioconductor, and we prefer to use [`https://www.tidyverse.org/`](https://www.tidyverse.org/) for `tidyverse` functions.
 
-+ Be on a MacOS computer
-+ Install and/or open the [MacDown program](https://macdown.uranusjr.com/)
-+ Open the markdown cheatsheet in MacDown, and go to `File` -> `Export` -> `PDF`
-+ Save appropriately and voila!
+Cheatsheets are written in plain markdown and are converted to a shareable PDF format using the `Node.js` package [`mdpdf`](https://github.com/BlueHatbRit/mdpdf), with the default PDF style.
 
-When choosing documentation links to incorporate in cheatsheets, we prefer to use [`https://rdrr.io/`](https://rdrr.io/) when possible for Base R and Bioconductor, and we prefer to use [`https://www.tidyverse.org/`](https://www.tidyverse.org/) for `tidyverse` functions. 
\ No newline at end of file
+To render these packages, you will therefore have to first install `npm`, the `Node.js` package manager.
+You can install `Node.js` and `npm` using Homebrew with `brew install node`, or you can install into a Conda environment with `conda install nodejs`
+With `Node.js` installed, you can install the `mdpdf` package into the default `Node.js` library with:
+
+```
+npm install -g mdpdf
+```
+
+In addition, cheatsheet table of contents are created with the `Node.js` library [`doctoc`](https://github.com/thlorenz/doctoc), which can similarly be globally installed with:
+
+```
+npm install -g doctoc
+```
+
+To re-render a cheatsheet to PDF after making desired changes in its markdown, take the following steps:
+
+* Navigate in terminal to the `module-cheatsheets` directory
+* Run `doctoc` on the file to update its table of contents: `doctoc cheatsheet-file.md`
+  * Alternatively, you can use a tool such as the VS Code Extension [Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one).
+  If you choose this route, ensure that the table of contents is not duplicated.
```

Comment:
I'm pretty sure the way this happened is that Markdown All in One does not use HTML comment delimiters and instead identifies the TOC through simple heuristics. So I probably created a TOC using that, so there were no HTML comments, then somebody ran `doctoc` which didn't find its comments, so created a new TOC.

Reply:
>  then somebody ran `doctoc` which didn't find its comments, so created a new TOC.

Ah yes! It was me 😬